### PR TITLE
LPS-161202 let properties task can load content from gradle-local.properties

### PR DESCRIPTION
**Pull request by simonjhy**
@drewbrokke Hi drew, when I did lps-161202 and run gradleTest, I found the several productInfo gradleTests canp not pass. I found the net.saliman.properties plugin need to apply after workspaceExtension initilaized, so I changed its from code line 64 to 67. The chagnes passed all gradleTest now.

### CODE REVIEW

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspacePlugin.java [hunk 1]
```diff
@@ -61,11 +61,11 @@ public void apply(Settings settings) {
 		Gradle gradle = settings.getGradle();
 		File rootDir = settings.getRootDir();
 
-		_applyPlugins(settings);
-
 		final WorkspaceExtension workspaceExtension = _addWorkspaceExtension(
 			settings);
 
+		_applyPlugins(settings);
+
```
Comment (drewbrokke) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspacePlugin.java:68]: @simonjhy why does this fix the test cases? What was broken about them?

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java [hunk 2]
```diff
@@ -1863,6 +1869,67 @@ public void execute(
...
+										key.toString(), properties.get(key));
+								}
+							}
+
+						});
+				}
+
+			});
+	}
+
```
Comment (drewbrokke) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java:1932]: @simonjhy the purpose of the plugin is to read those properties, we should not have to do this manually. We need to look into why the properties task is not reading the override properties found in the `gradle-local.properties` file.

My guess is that the plugin needs to be applied to projects, not just the settings.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Hey @simonjhy thank you for sending this change. I think we need to find a different solution - if the properties task is missing the override properties, it's possible other places are missing them as well.

In any case, I do not think that manually reading and copying the properties is the right approach in this case, since the plugin should already be handling that for us. Otherwise, what is the point of having the plugin?

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java (hunk 2 above)
Comment (simonjhy) [reply modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java:1932]: OK. got it.

**Pull request closed without merging**